src/main/kotlin/com/exactpro/th2/common/schema/message/MessageRouterUtils.kt

```diff
@@ -56,17 +53,4 @@ fun MessageRouter<EventBatch>.storeEvent(
     }
 
     storeEvent(this, parentId)
-}
-
-// Not using default parameters for java compatibility
-@Deprecated(message = "Only for back compatibility", replaceWith = ReplaceWith("toJson(true)", imports = ["com.exactpro.th2.common.schema.message.toJson"]), level = DeprecationLevel.WARNING)
-fun MessageOrBuilder.toJson() : String = this.toJson(true)
```

Comment:
We should keep, but deprecate this function instead, to avoid breaking projects whom are already using it